src/ejabberd_c2s.erl

```diff
@@ -861,17 +859,16 @@ process_outgoing_stanza(Acc, StateData) ->
     ToJID = mongoose_acc:to_jid(Acc),
     Element = mongoose_acc:element(Acc),
     NState = process_outgoing_stanza(Acc, ToJID, Element#xmlel.name, StateData),
-    ejabberd_hooks:run(c2s_loop_debug, [{xmlstreamelement, Element}]),
```

Comment:
are we getting read of `c2s_loop_debug` hook?

Reply:
Yes we do, unless there is strong opinion not to do so. I don't remember any situation when it was in fact used.